resources/js/src/app/components/customer/AddressSelect/AddressSelect.js

```diff
@@ -59,8 +61,6 @@ Vue.component("address-select", {
          */
         addEventListener()
         {
-            var self = this;
-
             ApiService.listen("AfterAccountContactLogout",
```

Comment:
mach das hier über eine arrow function 

ApiService.listen("AfterAccountContactLogout", () => {
                     this.cleanUserAddressData();
                 });